lib/IRGen/GenMeta.cpp

```diff
@@ -3789,10 +3806,34 @@ namespace {
 
       return flags;
     }
+  };
+
+  // FIXME: Without this template typealias, the following errors are produced 
```

Comment:
Interesting, is there a radar for this?

Reply:
Unfortunately, I wasn't able to minimally recreate the problem, but rdar://problem/58884416 covers this.